Process.c

```diff
@@ -443,7 +446,17 @@ void Process_writeField(Process* this, RichString* str, ProcessField field) {
    }
    case PROCESSOR: xSnprintf(buffer, n, "%3d ", Settings_cpuId(this->settings, this->processor)); break;
    case SESSION: xSnprintf(buffer, n, Process_pidFormat, this->session); break;
-   case STARTTIME: xSnprintf(buffer, n, "%s", this->starttime_show); break;
+   case STARTTIME: {
+#ifdef __linux__
```

Comment:
This should be moved to `linux/LinuxProcess.c`, where it will override the one from `Process.c`.

In general, code at the root directory files should be portable, and not have any `ifdef __linux__` in them.